TheWarWithin/HunterBeastMastery.lua

```diff
@@ -1245,8 +1295,56 @@ end )
 local CallOfTheWildCDR = setfenv( function()
     gainChargeTime( "kill_command", spec.abilities.kill_command.recharge/2)
     gainChargeTime( "barbed_shot", spec.abilities.barbed_shot.recharge/2)
+    if talent.withering_fire.enabled then applyBuff( "deathblow" ) end
 end, state )
 
+local pack_leader__buff_cycle = {
+    "howl_of_the_pack_leader_wyvern",
+    "howl_of_the_pack_leader_pig",
+    "howl_of_the_pack_leader_bear",
+}
+
+local pack_leader_buff_current = 1
```

Comment:
This needs to be a RegisterSpecExpr and have it get initialized from the value you're tracking in CLEU.  And reset this to `nil` on reset.  (Like the other thing I just noted.)